.github/workflows/docker-publish.yml

```diff
@@ -32,7 +32,6 @@ jobs:
         env: 
           pkg-version: ${{steps.previoustag.outputs.tag}}
         run: | 
-          cd oracle-server-ui
           docker build -t bitcoinscala/oracle-server-ui:latest -t bitcoinscala/oracle-server-ui:${{steps.previoustag.outputs.tag}}-${{ steps.vars.outputs.sha_short }}-SNAPSHOT .
```

Comment:
I think the `...-SNAPSHOT .` needs to change to `...-SNAPSHOT oracle-server-ui/` since we are no longer in the directory `oracle-server-ui`

Reply:
Alternatively you can move the `Dockerfile` to the root directory if you want